userguide/one_shot_object_detection/README.md

```diff
@@ -82,7 +72,9 @@ Examples of test image predictions:
 
 #### How it works
 
-To begin training for one-shot object detector, a set of synthetic training data needs to be generated using the provided starter training image(s). These images are augmented by applying yaw, pitch, and roll rotations. These operations are applied on a set of background images. The background images may be provided by the user. If no background images are provided, a set of default background images are automatically downloaded and used.
+To begin training a one-shot object detector, a set of synthetic training data needs to be generated using the provided starter training image(s). The synthetic images are generated by applying yaw, pitch, and roll rotations on a set of background images. The background images may be provided by the user. If no background images are provided, a set of default background images are automatically downloaded and used.
```

Comment:
I think you need to simplify this explanation. 

In this section, we will go into detail on what happens under the hood of the create method. An OSOD model is trained in 2 steps. First, we use the starter images to generate synthetic data and annotations, and second, we train an object detector.

Then go into a short paragraph explaining how the synthetic images are generated.